drivers/src/soc_ifc.rs

```diff
@@ -131,6 +132,85 @@ impl SocIfc {
         let flags: MfgFlags = soc_ifc_regs.cptra_dbg_manuf_service_reg().read().into();
         flags.contains(MfgFlags::GENERATE_IDEVID_CSR)
     }
+
+    /// Enbale or disable WDT1
+    ///
+    /// # Arguments
+    /// * `enable` - Enable or disable WDT1
+    ///
+    pub fn configure_wdt1(&mut self, enable: bool) {
```

Comment:
Don't you need to setup the interrupt handling somewhere?

Reply:
Looking at the docs, it looks like an NMI will eventually be generated if we don't respond to the original interrupt; maybe that's good enough.

Reply:
I'd still like to see this thing actually fire in verilator to make sure we're doing this right.

Reply:
Not handling WDT1 expiration interrupt causes the WDT2 timer to be "enabled'. Subsequently, not handling WDT2 expiration interrupt triggers an NMI.